### PR TITLE
I did too many things

### PR DESCRIPTION
**Pull request by uuunwired**
I did too many things.

**added:**
- Lightroom Mobile (com.adobe.lrmobile)
- Roblox (com.roblox.client)
- Infinity for Reddit (ml.docilealligator.infinityforreddit)
- Capcut (com.lemon.lvoverseas)
- Samsung Health (com.sec.android.app.shealth)

**linked:**
- Aliucord (com.aliucord) -> Discord
- PhotonCamera (com.particlesdevs.photoncamera) -> Camera
- Shopee (com.shopee.id, com.shopee.my, com.shopee.th, com.shopee.vn, com.shopee.tw, com.shopee.br) -> Shopee (btw shopee exists in countries other than the Philippines) 
- LOS/AOSP, MIUI and OOS messages (com.android.messaging, com.android.mms, com.android.sms, com.oneplus.mms) -> Messages

### CODE REVIEW

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
Thanks for your interest in contributing! For the CapCut icon, I’d suggest making the logo a bit larger. For the Infinity for Reddit icon, how about separating the UFO from its driver? These suggestions are reflected below.

File: app/src/main/res/drawable/infinity_for_reddit.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      android:fillType="evenOdd"
+      android:strokeLineCap="round"/>
+  <path
+      android:pathData="m3.13192,4.10326c-0.47337,0 -1.02242,-0.09241 -1.61139,-0.36564l-0.0088,-0.00411 -0.00078,-0.00966C1.51055,3.71924 1.47516,3.26069 1.65643,2.7588 1.7898,2.38974 2.06509,1.93629 2.63304,1.66027 3.09347,1.43649 3.64594,1.48017 4.04386,1.80222c0.01398,0.01138 0.02806,0.02313 0.04194,0.0353 0.42461,0.37083 0.66099,0.9921 0.70268,1.84652l0.00046,0.00983 -0.00781,0.00519c-0.00979,0.00605 -0.66606,0.40424 -1.64867,0.40424z"
+      android:strokeAlpha="0.974359"
+      android:strokeWidth="0.13229167"
+      android:fillColor="#00000000"
+      android:strokeColor="#000000"
+      android:fillType="evenOdd"/>
+</vector>
```
[the remark this answers is not in the document]
Comment (uuunwired) [reply app/src/main/res/drawable/infinity_for_reddit.xml]: Added the suggested changes to my repo. Thanks.

**Comment by uuunwired** [comment]
**Update 12/27**

**added:**
* AliExpress (com.alibaba.aliexpresshd)
* Lazada (com.lazada.android)
* Citra Emulator (org.citra.citra_emu)
* Dolphin Emulator (org.dolphinemu.dolphinemu)
* DraStic DS Emulator (com.dsemu.drastic)
* Pokemon GO (com.nianticlabs.pokemongo)
* Viber (com.viber.voip)

**Comment by uuunwired** [comment]
**Update 12/29**

**added:**

* 3DMark (com.futuremark.dmandroid.application)
* AIDA64 (com.finalwire.aida64)
* AndroBench (com.andromeda.androbench2)
* AppChecker (com.kroegerama.appchecker)
* Canva (com.canva.editor)
* Device care (Samsung) (com.samsung.android.lool)
* Device Info HW (ru.andr7e.deviceinfohw)
* DiskInfo (me.kuder.diskinfo)
* Geekbench (com.primatelabs.geekbench, com.primatelabs.geekbench5)
* Geometric Weather (wangdaye.com.geometricweather)
* Goodreads (com.goodreads)
* GSam Battery Monitor (com.gsamlabs.bbm)
* Network Cell Info (Lite) (com.wilysis.cellinfo, com.wilysis.cellinfolite)
* Phone Info SAM (org.vndnguyen.phoneinfo)
* QuickSwitch (xyz.paphonb.quickstepswitcher)
* Screen Recorder (Kimcy929) (com.kimcy929.screenrecorder)
* Shizuku (moe.shizuku.privileged.api)
* YASNAC (rikka.safetynetchecker)
* YouTube Studio (com.google.android.apps.youtube.creator)
* Zoom (us.zoom.videomeetings)

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
Iʼd suggest some minor sizing tweaks: [suggestions.zip](https://github.com/LawnchairLauncher/lawnicons/files/7876850/suggestions.zip).

**Comment by ZhmieXD** [comment]
Oh wow turns out some of the icons I'm currently doing are currently in this pull request, Mind if I resolve the issue?

**Comment by uuunwired** [comment]
> Oh wow turns out some of the icons I'm currently doing are currently in this pull request, Mind if I resolve the issue?

Sure go ahead!

**Comment by ZhmieXD** [comment]
After a rest ^^;

**Comment by ZhmieXD** [comment]
Alright, so, can the suggestions be a bit more specific? @patrykmichalik

**Comment by ZhmieXD** [comment]
diskinfo
drastic_ds_emulator

Here, I uploaded the tweaked svg files requested in the suggestions. hope this is enough ^^;

**Comment by patrykmichalik** [comment]
Just to clarify, my comment above includes a ZIP file with some minor tweaks that I’d suggest. These changes would have to be applied by @uuunwired.

**Comment by ZhmieXD** [comment]
> Just to clarify, my comment above includes a ZIP file with some minor tweaks that I’d suggest. These changes would have to be applied by @uuunwired.

I know, it's all on him to apply it.
But does that means you already put your modified version of it? Alright then 👍

**Comment by Chefski** [comment]
Thanks for all these additions. Unfortunately I'll be closing this as the icons are outdated.

**Pull request closed without merging**